src/PullCountTagSelect.tsx

```diff
@@ -34,6 +36,17 @@ export class PullCountTagSelect extends React.Component<IPullCountTagSelectProps
     }
   }
 
+  componentDidUpdate() {
+    const tags = this.getTags();
+    if (tags.length > 0 && !this.state.initialized) {
+      const selected = tags.length === 1
+                  ? tags
+                  : tags[0] === 'latest' ? tags.slice(1, 3) : tags.slice(0, 2);
```

Comment:
It seems a little brittle to assume that `latest` would only ever appear in the first index if it does. Can you think of something a little more robust so that the sort order of the tags wouldn't affect this?

Reply:
True, changed the section.